tests/unit/styleMixTests.cpp

```diff
@@ -13,6 +13,54 @@
 using namespace Tangram;
 using YAML::Node;
 
+TEST_CASE( "Style Mixing Test: Nested Style Mixin Nodes", "[mixing][core][yaml]") {
+    SceneLoader sceneLoader;
+    Mixes mix;
+    Node mixNode;
+    Node node = YAML::Load(R"END(
```

Comment:
I have a test where a style is mixed out of order: https://github.com/tangrams/tangram-es/pull/302/files#diff-f021195bbca063598e222f41b7a41950R29